scripts/populate_preprint_providers.py

```diff
@@ -96,7 +96,8 @@ def main(env):
             'social_instagram': '',
             'licenses_acceptable': ['CC0 1.0 Universal', 'CC-By Attribution 4.0 International', 'No license'],
             'header_text': '',
-            'subjects_acceptable': [],
+            'share_source': 'OSF' if env == 'prod' else 'OSF-{}'.format(env),
```

Comment:
This file should be going away in your other PR, so probably ditch the changes in here.